projects/components/src/filtering/filter-button/filter-button.component.ts

```diff
@@ -60,13 +63,21 @@ export class FilterButtonComponent implements OnChanges {
   public ngOnChanges(): void {
     this.availableFilters =
       this.attribute !== undefined && this.value !== undefined
-        ? this.filterBuilderLookupService
-            .lookup(this.attribute.type)
-            .buildFiltersForSupportedOperators(this.attribute, this.value)
+        ? this.buildAvailableFilters(this.attribute, this.value, this.subpath)
         : [];
   }
 
   public onFilterClick(filter: Filter): void {
     this.filterUrlService.addUrlFilter(this.metadata || [], filter);
   }
+
+  private buildAvailableFilters(attribute: FilterAttribute, value: FilterValue, subpath?: string): Filter[] {
+    if (subpath !== undefined) {
```

Comment:
Push this down - filter builder should handle the optional subpath.

Reply:
done